### ISSUE TITLE
Unused uses false positive

### ISSUE THREAD

**Issue by josephzidell** [slevomat/coding-standard#575]
UnusedUsesSniff reports a false positive in the below scenario.

```php
use MyClass;

/**
 * @return MyClass
 */
function foo() {}
```

**Comment by kukulich** [comment]
See readme.
